pkg/argocd/argocd.go

```diff
@@ -103,13 +105,20 @@ func Login(opts *LoginOptions) error {
 		"--port-forward",
 		"--port-forward-namespace",
 		opts.Namespace,
-		"--password",
-		opts.Password,
 		"--username",
 		opts.Username,
+		"--password",
+		opts.Password,
 		"--name",
 		"autopilot",
 	}
+	if opts.KubeConfig != "" {
+		origKubeConfig := os.Getenv("KUBECONFIG")
+		defer func() { os.Setenv("KUBECONFIG", origKubeConfig) }()
+		if err := os.Setenv("KUBECONFIG", opts.KubeConfig); err != nil {
+			return err
```

Comment:
return a wrapped error here: `failed to set KUBECONFIG env var`, cause I'm not sure the error returned here would be clear